app/assets/javascripts/discourse/app/templates/full-page-search.hbs

```diff
@@ -50,6 +50,8 @@
         {{search-advanced-options
           searchTerm=(readonly searchTerm)
           onChangeSearchTerm=(action (mut searchTerm))
+          search=(action "search" true)
```

Comment:
I know you didn't introduce this, but I dislike boolean parameters like this that aren't named. I have no idea what `true` represents from reading it. Maybe if we have time we can circle back and changed it to a `something=true` parameter?